rts/lra/lra-coordinator/pom.xml

```diff
@@ -61,6 +61,10 @@
     </build>
 
     <dependencies>
+        <dependency>
```

Comment:
Not used

Reply:
no, this is used for getting JAX-RS client in the coordinator.

Reply:
Maybe this is a runtime dependency (although not marked as such) so I will resolve this conversation.